desktop/src/app.rs

```diff
@@ -211,7 +213,9 @@ impl App {
                                 MouseButton::Left => RuffleMouseButton::Left,
                                 MouseButton::Right => RuffleMouseButton::Right,
                                 MouseButton::Middle => RuffleMouseButton::Middle,
-                                MouseButton::Other(_) => RuffleMouseButton::Unknown,
+                                MouseButton::Back
+                                | MouseButton::Forward
+                                | MouseButton::Other(_) => RuffleMouseButton::Unknown,
```

Comment:
Is it nicer to list all these variants instead of just `_`? I don't mind, and it's not that many lines, just caught my eye.

Reply:
Wasn't sure if we'd want to be informed on new enums or not :D

Reply:
Fair point, it's fine like this. (Although, _how many_ additional "named" buttons could a mouse gain?)

Reply:
Decided to go for `_` :D that's a fair point and I can't argue it.